### DISCUSSION TITLE
Using Pandas as alternative numpy arrays

### DISCUSSION THREAD

**Discussion by nunobrum** [nunobrum/PyLTSpice#76] [Polls]
Pandas library has become a very popular library for treating data.
I'm wondering if it would be preferred to the numpy array to store data.
The advantages I can see are:

- Possibility of exporting to many other formats and use in other libraries that use pandas (ex: plotly, sklearn)
- All traces are stored in the same object.
- Quite powerful filtering and access to data

**Comment by mvanriet** [comment]
Isn't numpy more powerful for processing numerical data ?
E.g. for calculating an FFT.
Also scipy uses numpy arrays.

Maybe rather a method to convert (parts of) RawRead data to a pandas object?
